cli/ops/plugin.rs

```diff
@@ -69,6 +75,7 @@ pub fn op_open_plugin(
   Ok(JsonOp::Sync(json!(rid)))
 }
 
+/// A resource that stores a dynamically loaded plugin.
```

Comment:
👍 This should help avoid any confusion between this and `PluginDefinedResource`.